src/style.css

```diff
@@ -1 +1,58 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  /* align-items: flex-start; */
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin-top: 10px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.link {
+  display: flex;
+  align-items: center;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  position: relative;
+  height: 60px;
+  margin-left: 20px;
+}
+
+.is-active {
+  color: rgba(0, 172, 220, 1);
+  margin-left: 0;
+}
```

Comment:
here you reset the margin, you do that so that the first element does not have any margin, but is-active element and the first element don't always have to be the same. what if I make the second element active and not the first one. you want to reset the margin for the first element, so use :first-child pseudo-class